.buildkite/rllib_contrib.rayci.yml

```diff
@@ -2,6 +2,9 @@ group: rllib contrib tests
 depends_on:
   - oss-ci-base_build
 steps:
+  - block: "run rllib contrib tests"
+    if: build.env("BUILDKITE_PIPELINE_ID") == "0189942e-0876-4b8f-80a4-617f988ec59b" && ( build.branch != "master" && build.branch !~ /^releases\/.*/ )
```

Comment:
I do not think we really run tests on premerge with release branches but anyways..